asttokens/util.py

```diff
@@ -142,6 +142,11 @@ def is_joined_str(node):
   return node.__class__.__name__ == 'JoinedStr'
 
 
+def is_starred(node):
+  """Returns whether node is an expression node."""
```

Comment:
This comment looks copy-pasted from elsewhere and a bit off.

Reply:
```suggestion
  """Returns whether node is a starred expression node."""
```